pkg/adapter/nimbus-kyverno/manager/manager.go

```diff
@@ -382,3 +413,28 @@ func deleteDanglingkcps(ctx context.Context, cnp v1alpha1.ClusterNimbusPolicy, l
 
 	}
 }
+
+func createTriggerForKp(ctx context.Context, namespace string) {
+	logger := log.FromContext(ctx)
+	configMap := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "kyverno-trigger-mutate-existing-configmap",
+			Namespace: namespace,
```

Comment:
Add a owner reference for this configmap. Set the Kp mutating policy as the owner policy so that when the KP is deleted , the config map is also deleted